molecularnodes/blender/nodes.py

```diff
@@ -237,6 +250,16 @@ def material_default():
 
     return bpy.data.materials[mat_name]
 
+def MN_micrograph_material():
```

Comment:
This currently doesn't append the material?

Reply:
additionally I think it could be renamed to `material_micrograph()` to align with `material_default()`

Reply:
This gets imported automatically when the MN_starfile_instances nodetree gets loaded, since it's being used there. I had some loading code there, but (I) the testcoverage report complained that this is never executed and (II) I would have to replicate that logical to import from the right data file. So I just deleted it and things work fine. 

This material should be somewhat hidden from inexperienced users anyway, as it get modified by events, which might have hard to understand consequences.

Reply:
that's fair. yeah it's annoying with the test coverage not liking certain aspects like the UI components that are hard to test.